packages/postgres-database/src/simcore_postgres_database/models/groups.py

```diff
@@ -0,0 +1,127 @@
+""" Groups table
+
+    - List of groups in the framework
+    - Groups have a ID, name and a list of users that belong to the group
+"""
+
+import enum
+
+import sqlalchemy as sa
+from sqlalchemy.sql import func
+
+from .base import metadata
+
+
+class GroupType(enum.Enum):
+    """
+        standard: standard group
```

Comment:
can you define a bit more what is the different between standard and primary here?

Reply:
yes done, basically this follows linux mechanism.
a primary group is the group created with the user. the user is typically the only one within that group. the primary group cannot be deleted unless the user is deleted.